Makefile.one

```diff
@@ -0,0 +1,468 @@
+	@echo "CMD_MKDIR                $(CMD_MKDIR)"
+	@echo "CMD_OPA                  $(CMD_OPA)"
+	@echo "CMD_PKGCONFIG            $(CMD_PKGCONFIG)"
+	@echo "CMD_RM                   $(CMD_RM)"
+	@echo "CMD_STRIP                $(CMD_STRIP)"
+	@echo ---------------------------------------
+	@echo "LIB_ELF                  $(LIB_ELF)"
+	@echo "LIB_ZLIB                 $(LIB_ZLIB)"
+	@echo ---------------------------------------
+	@echo "VERSION                  $(VERSION)"
+	@echo "LAST_GIT_TAG             $(LAST_GIT_TAG)"
+	@echo "BPF_NOCORE_TAG           $(BPF_NOCORE_TAG)"
+	@echo ---------------------------------------
+	@echo "UNAME_M                  $(UNAME_M)"
+	@echo "UNAME_R                  $(UNAME_R)"
+	@echo "ARCH                     $(ARCH)"
+	@echo "LINUX_ARCH               $(LINUX_ARCH)"
+	@echo ---------------------------------------
+	@echo "OUTPUT_DIR               $(OUTPUT_DIR)"
+	@echo ---------------------------------------
+	@echo "KERN_RELEASE             $(KERN_RELEASE)"
+	@echo "KERN_BUILD_PATH          $(KERN_BUILD_PATH)"
+	@echo "KERN_SRC_PATH            $(KERN_SRC_PATH)"
+	@echo ---------------------------------------
+	@echo "LIBBPF_CFLAGS            $(LIBBPF_CFLAGS)"
+	@echo "LIBBPF_LDLAGS            $(LIBBPF_LDFLAGS)"
+	@echo "LIBBPF_SRC               $(LIBBPF_SRC)"
+	@echo ---------------------------------------
+	@echo "STATIC                   $(STATIC)"
+	@echo ---------------------------------------
+	@echo "BPF_VCPU                 $(BPF_VCPU)"
+	@echo "TRACEE_EBPF_OBJ_SRC      $(TRACEE_EBPF_OBJ_SRC)"
+	@echo ---------------------------------------
+	@echo "GO_ARCH                  $(GO_ARCH)"
+	@echo "GO_TAGS_EBPF             $(GO_TAGS_EBPF)"
+	@echo "GO_TAGS_RULES            $(GO_TAGS_RULES)"
+	@echo ---------------------------------------
+	@echo "CUSTOM_CGO_CFLAGS        $(CUSTOM_CGO_CFLAGS)"
+	@echo "CUSTOM_CGO_LDFLAGS       $(CUSTOM_CGO_LDFLAGS)"
+	@echo "CGO_EXT_LDFLAGS_EBPF     $(CGO_EXT_LDFLAGS_EBPF)"
+	@echo "CGO_EXT_LDFLAGS_RULES    $(CGO_EXT_LDFLAGS_RULES)"
+	@echo ---------------------------------------
+	@echo "GO_ENV_EBPF              $(GO_ENV_EBPF)"
+	@echo "GO_ENV_RULES             $(GO_ENV_RULES)"
+	@echo ---------------------------------------
+	@echo "TRACEE_EBPF_SRC          $(TRACEE_EBPF_SRC)"
+	@echo ---------------------------------------
+	@echo "GOSIGNATURES_DIR         $(GOSIGNATURES_DIR)"
+	@echo "GOSIGNATURES_SRC         $(GOSIGNATURES_SRC)"
+	@echo ---------------------------------------
+	@echo "REGO_SIGNATURES_DIR      $(REGO_SIGNATURES_DIR)"
+	@echo "REGO_SIGNATURES_SRC      $(REGO_SIGNATURES_SRC)"
+	@echo ---------------------------------------
+
+# usage
+
+.PHONY: help
+help:
+	@echo ""
+	@echo "$ make all                  # build tracee-ebpf, tracee-rules & rules"
+	@echo ""
+	@echo "$ make env                  # show makefile environment/variables"
+	@echo "$ make reqs                 # verify build requirements"
+	@echo ""
+	@echo "# build"
+	@echo ""
```

Comment:
You can move:
`@echo "$ make all # build tracee-ebpf, tracee-rules & rules"`
Here, which I think makes more sense.